libs/shared/src/lib/components/widgets/summary-card/summary-card-item-content/summary-card-item-content.component.ts

```diff
@@ -134,6 +134,7 @@ export class SummaryCardItemContentComponent
           [filterField]: cleanFilterValue,
         }),
       };
+      this.contextService.lastComponentEmitter = this.parent.parent;
```

Comment:
```suggestion
      this.contextService.lastComponentEmitter = this.parent?.parent;
```
If parent is optional we should check property just in case 😉